### PR TITLE
[docs] Add some content to the landing page.

### PR DESCRIPTION
**Pull request by wkozyra95**
To run that locally go to docs directory and run
```
cargo run --bin generate_wiki && npm install && npm start
```

20231201_14h00m44s_grim

### CODE REVIEW

File: docs/src/pages/index.tsx [hunk 1]
```diff
@@ -1,37 +1,259 @@
...
+
+  return (
+    <div className="container margin-top--lg margin-bottom--md">
+      <Heading as="h2" className="margin-bottom--md">
+        Features
+      </Heading>
+
+      <Feature
+        text="WebGPU APIs"
+        secondaryText="Use existing WebGPU-based components provided by the compositor. If you need anything not covered by the existing components, you can always write your own WGSL shader that implements that effect."
```
Comment (noituri) [inline docs/src/pages/index.tsx:177]:
```suggestion
        secondaryText="Utilize existing WebGPU-based components provided by the compositor or craft your own WGSL shader to achieve any desired effect."
```

IMO making it shorter makes the description more interesting

File: docs/src/pages/index.tsx [hunk 2]
```diff
@@ -1,37 +1,259 @@
...
+      />
+      <Feature
+        text="Real-time processing"
+        secondaryText="Process live video sources in real-time with low latency."
+        image={<MdStream className="margin--lg" style={{ fontSize: 100 }} />}
+        inverted
+      />
+      <Feature
+        text="Static content"
+        secondaryText="Render text and static images onto your scene or pass them to other components for further processing. Currently we support: PNG, JPEG, SVG, and GIFs."
```
Comment (noituri) [inline docs/src/pages/index.tsx:188]:
```suggestion
        secondaryText="Render text and static images onto your scene or pass them to other components for further processing."
```

I would put that information in the docs instead

**Review by noituri — COMMENTED** [review]
image

It feels like something is missing here. Maybe use larger images or/and fonts?

File: docs/src/pages/index.tsx [hunk 3]
```diff
@@ -1,37 +1,259 @@
...
+            The entire configuration process happens over HTTP API, and streams are delivered over
+            RTP. You can leverage technology of your choice and easily integrate it with your own
+            solutions.
+          </p>
+        </ProsCard>
+        <ProsCard title="Hardware accelerated" icon={FaMicrochip}>
+          <p className="padding--md">
+            Rendering is GPU accelerated using {wgpuLink} library, which implements API based on the
+            WebGPU standard. Parts of the standard, like the ability to run WGSL shaders, are
+            directly accessible via API.
```
Comment (WojciechBarczynski) [inline docs/src/pages/index.tsx:87]: Rendering is GPU accelerated using Rust {wgpuLink} library, based on the WebGPU standard. 
Low-level integrations, like custom WGSL shaders, are directly accessible via LiveCompositor API.

File: docs/src/pages/index.tsx (hunk 1 above)
Comment (WojciechBarczynski) [inline docs/src/pages/index.tsx:177]:
```suggestion
        secondaryText="
        Utilize existing WebGPU-based components or craft your own WGSL shader to achieve any desired effect and directly leverage GPU parallel processing capabilities.
        "
```

File: docs/src/pages/index.tsx [hunk 4]
```diff
@@ -1,37 +1,259 @@
...
+      </Heading>
+
+      <Feature
+        text="WebGPU APIs"
+        secondaryText="Use existing WebGPU-based components provided by the compositor. If you need anything not covered by the existing components, you can always write your own WGSL shader that implements that effect."
+        image={wgpuLogo}
+      />
+      <Feature
+        text="Real-time processing"
+        secondaryText="Process live video sources in real-time with low latency."
```
Comment (WojciechBarczynski) [inline docs/src/pages/index.tsx:182]:
```suggestion
        secondaryText="
        Process live video streams in real-time with low latency. 
        Blazingly fast processing makes video conferencing, live-streaming and broadcasting an everyday bread and butter for LiveCompositor.
        "
```

File: docs/src/pages/index.tsx [hunk 5]
```diff
@@ -1,37 +1,259 @@
...
+        image={<MdStream className="margin--lg" style={{ fontSize: 100 }} />}
```
Comment (WojciechBarczynski) [inline docs/src/pages/index.tsx:183]:
```suggestion
        image={<MdLiveTv className="margin--lg" style={{ fontSize: 100 }} />}
```

File: docs/src/pages/index.tsx [hunk 6]
```diff
@@ -1,37 +1,259 @@
...
+        inverted
+      />
+      <Feature
+        text="Static content"
+        secondaryText="Render text and static images onto your scene or pass them to other components for further processing. Currently we support: PNG, JPEG, SVG, and GIFs."
+        image={imageTextRendering}
+      />
```
Comment (WojciechBarczynski) [inline docs/src/pages/index.tsx:190]: I would avoid using `scene` here
```suggestion
      <Feature
        text="Static content"
        secondaryText="Render text and static images onto your output streams or pass them to other components for further processing. Currently we support: PNG, JPEG, SVG, and GIFs."
        image={imageTextRendering}
      />
```

File: docs/src/pages/index.tsx [hunk 7]
```diff
@@ -1,37 +1,259 @@
...
+        <div className="card__header">
+          <Heading as="h3" className="margin-top--sm">
+            Membrane plugin
+          </Heading>
+        </div>
+        <div className="card__body container">
+          <p>
+            Membrane is a developer-friendly multimedia framework for Elixir. You can easily add
+            video composing functionality into your multimedia pipeline using Live Compositor
+            plugin.
```
Comment (WojciechBarczynski) [inline docs/src/pages/index.tsx:215]: It's a proper name, so use `Membrane Live Compositor Plugin` everywhere
```suggestion
            Membrane is a developer-friendly multimedia framework for Elixir. You can easily add
            video composing functionality into your multimedia pipeline using Live Compositor
            Plugin.
```

File: docs/src/pages/index.tsx
Comment (WojciechBarczynski) [inline docs/src/pages/index.tsx:1]: IDK where to add this suggestion, but I think navbar should have docs link as well

File: docs/src/pages/index.tsx (hunk 3 above)
Comment (wkozyra95) [reply docs/src/pages/index.tsx:87]: >Rendering is GPU accelerated using Rust {wgpuLink} library, based on the WebGPU standard.

I don't think this sentence is clear, what is based on WebGPU? rendering? wgpu?

> Low-level integrations

shaders are not low level parts of the standard, also not sure what integrations here is supposed to mean

File: docs/src/pages/index.tsx
Comment (wkozyra95) [reply docs/src/pages/index.tsx:1]: If we would have any other tab than docs then I agree, but I we only have Docs tab and main page it looks wierd

File: docs/src/pages/index.tsx [hunk 8]
```diff
@@ -1,37 +1,259 @@
...
+                </Link>
+                <Link
+                  className="button button--secondary button--outline button--lg col col-6 margin--sm"
+                  to="https://github.com/membraneframework/video_compositor">
+                  View on GitHub
+                </Link>
+              </div>
+            </div>
+          </div>
+          <div className="col col--6">Image</div>
```
Comment (noituri) [inline docs/src/pages/index.tsx:54]: Is it just a temporary placeholder?

Comment (wkozyra95) [reply docs/src/pages/index.tsx:54]: 87wh9n

File: docs/src/pages/index.tsx
Comment (WojciechBarczynski) [reply docs/src/pages/index.tsx:1]: Maybe docs and github tabs?

**Review by WojciechBarczynski — APPROVED** [review]

**Pull request merged**